IntelliJ/Nx/build.gradle.kts

```diff
@@ -0,0 +1,176 @@
+plugins {
+    id("java")
+}
+
+group = "com.nuix.nx"
+version = "1.19.0-SNAPSHOT"
+
+val sourceCompatibility = 11
+val targetCompatibility = 11
+
+// Directory containing Nuix Engine release
+val nuixEngineDirectory: String = System.getenv("NUIX_ENGINE_DIR")
+println("NUIX_ENGINE_DIR: ${nuixEngineDirectory}")
+if (nuixEngineDirectory.isEmpty()) {
+    throw InvalidUserDataException("Please populate the environment variable 'NUIX_ENGINE_DIR' with directory containing a Nuix Engine release")
+}
+
+val engineLibDir = "${nuixEngineDirectory}\\lib"
+println("engineLibDir: ${engineLibDir}")
+
+val nuixAppLibDir = "C:\\Program Files\\Nuix\\Nuix 100.2\\lib"
```

Comment:
Design question: Is NX meant to be built by people outside Nuix?  Can be use the internal maven repo for engine libraries?  Will defer to what your plan is, but wanted to bring it up.

Reply:
It has traditionally been available.  Way back it was not, but customers asked for it as they learned of it so it has been public since putting on GitHub.  Probably will keep it that way.